src/middleware/authentication.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
Try and move these to the .eslintrc file and set them to "off" otherwise you end up with loads of these littered about